ui-app/client/src/components/PhoneNumber/phoneNumberProperties.ts

```diff
@@ -0,0 +1,424 @@
+		editor: ComponentPropertyEditor.ENUM,
+		group: ComponentPropertyGroup.BASIC,
+		enumValues: [
+			{ name: 'countryname', displayName: 'Country Name', description: 'Country Name' },
+			{ name: 'countrycode', displayName: 'Country Code', description: 'Country Code' },
+			{ name: 'dialcode', displayName: 'Dial Code', description: 'Dial Code' },
+		],
+	},
+	{
+		name: 'format',
+		schema: SCHEMA_BOOL_COMP_PROP,
+		displayName: 'Formar Number',
+		description: 'Format Phone Number.',
+		defaultValue: true,
+		group: ComponentPropertyGroup.BASIC,
+	},
+	{
+		name: 'storeFormatted',
+		schema: SCHEMA_BOOL_COMP_PROP,
+		displayName: 'Store Formatted Number',
+		description: 'Store the formatted phone number.',
+		defaultValue: false,
+		group: ComponentPropertyGroup.ADVANCED,
+	},
+	{
+		name: 'seperator',
+		schema: SCHEMA_STRING_COMP_PROP,
+		displayName: 'Format Value Type',
+		description: 'Type of the Value for Format',
+		defaultValue: ' ',
+		editor: ComponentPropertyEditor.ENUM,
+		group: ComponentPropertyGroup.BASIC,
+		enumValues: [
+			{ name: ' ', displayName: 'Space', description: 'Space' },
+			{ name: '-', displayName: 'Hyphen', description: 'Hyphen' },
+			{ name: '.', displayName: 'Dot', description: 'Dot' },
+		],
+	},
+	{
+		name: 'isSearchable',
+		schema: SCHEMA_BOOL_COMP_PROP,
+		displayName: 'Is Searchable',
+		description: 'Allows the users search options.',
+		defaultValue: false,
+		group: ComponentPropertyGroup.ADVANCED,
+	},
+	{
+		name: 'searchLabel',
+		schema: SCHEMA_STRING_COMP_PROP,
+		displayName: 'Search Label ',
+		description: 'Label for searchbox.',
+		translatable: true,
+		group: ComponentPropertyGroup.ADVANCED,
+	},
+	{
+		name: 'clearSearchTextOnClose',
+		schema: SCHEMA_BOOL_COMP_PROP,
+		displayName: 'Clear Search on close',
+		description: 'Clear Search on close.',
+		defaultValue: false,
+		group: ComponentPropertyGroup.ADVANCED,
+	},
+	{
+		name: 'noCodeForFirstCountry',
+		schema: SCHEMA_BOOL_COMP_PROP,
+		displayName: "Don't add dial code for first country",
+		defaultValue: false,
+		group: ComponentPropertyGroup.DATA,
+	},
+	{
+		name: 'noFloat',
+		schema: SCHEMA_BOOL_COMP_PROP,
+		displayName: 'Do not float Label',
+		description: 'Textbox without floating label.',
+		defaultValue: false,
+		group: ComponentPropertyGroup.ADVANCED,
+	},
+
+	{
+		name: 'autoComplete',
+		schema: SCHEMA_STRING_COMP_PROP,
+		displayName: 'Autocomplete type',
+		description:
+			'Autocomplete specifies what type or if any assistance that is provided to user while filling forms, uses same values as its html attribute.',
+		defaultValue: 'off',
+		group: ComponentPropertyGroup.ADVANCED,
+		enumValues: [
+			{ name: 'off', displayName: 'Off', description: 'Autocomplete off' },
+			{ name: 'on', displayName: 'On', description: 'Autocomplete on' },
+			{ name: 'none', displayName: 'None', description: 'No autocomplete' },
+		],
+	},
+
+	{
+		name: 'defaultValue',
+		schema: SCHEMA_ANY_COMP_PROP,
```

Comment:
Why is default value any type 
shouldn't it be string 
also Value state which this is being applied to inside the code is of type string 
also default value should be in group basic ?